packages/frontend-2/lib/dashboard/graphql/queries.ts

```diff
@@ -0,0 +1,13 @@
+import { graphql } from '~~/lib/common/generated/gql'
+
+export const dashboardProjectsQuery = graphql(`
+  query DashboardProjectsQuery {
```

Comment:
Call it `DashboardProjectsPageQuery` - the convention so far has been to add the Page bit in the name for all top-level page queries